### DISCUSSION TITLE
edge的侧边栏为什么不见了？

### DISCUSSION THREAD

**Discussion by Ofnoname** [viopsa233/AI-testers-Forum#47] [Q&A] [answered]
是被砍了，还是只针对中国用户不显示？

**Comment by NostalgiaGit** [comment, marked as answer]
是整个ai聊天框都不见了，还是只是没有ai聊天功能了？后者的话你需要挂魔法，现在应该本来就没对国内开放。前者的话去设置里弄一下，
![QQ截图20230514095243](https://github.com/viopsa233/AI-testers-Forum/assets/118115208/af5d0ddf-8b03-4f97-8092-9a4abf72956a)
![QQ截图20230514095306](https://github.com/viopsa233/AI-testers-Forum/assets/118115208/1d1b3944-10b6-484c-9193-b364152c7690)
![QQ截图20230514095320](https://github.com/viopsa233/AI-testers-Forum/assets/118115208/9e232a54-4e19-4eb9-8e66-9f6e3e77aeb9)

**Comment by Ofnoname** [comment]
谢谢 可以了
